app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -65,10 +66,26 @@
         app:data="@{viewModel.itemList}"
         app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
         app:layout_constraintBottom_toTopOf="@id/question_progress_bar"
-        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/center_guideline"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/end_session_body_text_view" />
 
+      <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/right_question_recycler_view"
```

Comment:
Perhaps: ``extra_interaction_recycler_view``?

Reply:
Done

Reply:
Resolving this on behalf of Ben.